plugins/com.google.cloud.tools.eclipse.appengine.ui/src/com/google/cloud/tools/eclipse/appengine/ui/AppEngineRuntime.java

```diff
@@ -22,11 +22,10 @@
  * Description of the App Engine runtime environments.
  */
 public enum AppEngineRuntime {
-  STANDARD_JAVA_7(Messages.getString("appengine.runtimes.java7"), null), //$NON-NLS-1$
```

Comment:
I don't think we should get rid of this yet.  But in hindsight this class is misguided.  IMHO we should instead be using the WTP IProjectFacet and decorating the App Engine facets with properties that can be queried.

Reply:
This is just an enum class whose values are used only in the project wizard UI and the `<runtime>` element in `appengine-web.xml` (this class is in the `.ui` bundle, and the Javadoc says "Description of the App Engine runtime environments"), so it should be fine and we can remove this. I guess you are thinking something else.

Reply:
👍  I had worried that it had spread beyond that.